<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlotTemplateWidget</class>
 <widget class="QWidget" name="PlotTemplateWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>786</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="lCaption">
       <property name="text">
        <string>Adding plot template</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbPreviousStep">
       <property name="text">
        <string>Previous step</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbAddTemplate">
       <property name="text">
        <string>Add template</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="PlotTemplateTreeWidget" name="twItems">
         <column>
          <property name="text">
           <string>1</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>2</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0,0,0,0,0,1">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Size</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cbSize">
             <property name="minimumSize">
              <size>
               <width>55</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="pbAddCursor">
           <property name="toolTip">
            <string>Cursor</string>
           </property>
           <property name="icon">
            <iconset resource="../../resource.qrc">
             <normaloff>:/resource/icons/compounds/add_polygon@2x.png</normaloff>:/resource/icons/compounds/add_polygon@2x.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbAddPen">
           <property name="toolTip">
            <string>Pen</string>
           </property>
           <property name="icon">
            <iconset resource="../../resource.qrc">
             <normaloff>:/resource/icons/compounds/add_curve@2x.png</normaloff>:/resource/icons/compounds/add_curve@2x.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbAddLine">
           <property name="toolTip">
            <string>Line</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../resource.qrc">
             <normaloff>:/resource/icons/compounds/add_line@2x.png</normaloff>:/resource/icons/compounds/add_line@2x.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbAddText">
           <property name="toolTip">
            <string>Text</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../resource.qrc">
             <normaloff>:/resource/icons/compounds/add_text@2x.png</normaloff>:/resource/icons/compounds/add_text@2x.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbAddImage">
           <property name="toolTip">
            <string>Image</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../resource.qrc">
             <normaloff>:/resource/icons/compounds/add_image@2x.png</normaloff>:/resource/icons/compounds/add_image@2x.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pbClearPlot">
             <property name="text">
              <string>Clear</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QSplitter" name="verticalSplitter">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="childrenCollapsible">
          <bool>false</bool>
         </property>
         <widget class="QSplitter" name="horizontalSplitter">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="childrenCollapsible">
           <bool>false</bool>
          </property>
          <widget class="GraphicsInteractiveModelPlot" name="wdtPlot" native="true"/>
          <widget class="QWidget" name="wdtHorizontalSpacer" native="true"/>
         </widget>
         <widget class="QWidget" name="wdtVerticalSpacer" native="true"/>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphicsInteractiveModelPlot</class>
   <extends>QWidget</extends>
   <header>ui/plots/graphics_interactive_model_plot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PlotTemplateTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>ui/widgets/plot_template_tree_widget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resource.qrc"/>
 </resources>
 <connections/>
</ui>
